### ISSUE TITLE
As a user I should be able to complete the setup by only using the keyboard

### ISSUE THREAD

**Issue by stripthis** [passbolt/passbolt_browser_extension#16]
As reported by Norman R.:

Some parts of the UI that are not currently accessible and HTML UI standards compliant (WCAG 2 Level AA) such as the show password being mouse only.

There seems to be some missing keyboard access and focus for the signup plugin process. I notice that someone has coded it properly for functional keyboard access in some parts (to download the plugin button) and yet missed it in other places. 

A clear visual focus instead of slight color change would also assist most users, (login.min.css line 10; opps, something went wrong 'retry' "button") including those in small screen (responsive design) and those with colorblindness.

**Comment by stripthis** [comment]
Fixed with v3
